script/Deploy.s.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
perhaps rename to DeployUSDC or DeployUSDCTokenBuyer ?

Reply:
done